editor/plugins/version_control_editor_plugin.cpp

```diff
@@ -30,47 +30,81 @@
 
 #include "version_control_editor_plugin.h"
 
-#include "core/object/script_language.h"
+#include "core/config/project_settings.h"
+#include "core/core_bind.h"
```

Comment:
I couldn't spot why it's included in the file but in general `core/core_bind.h` shouldn't be used outside core bindings. It's only meant to wrap some core functionality in a way that's convenient for GDScript / other scripting languages. But for C++ you should use the underlying core functionality directly.